app/routes/interestgroups/components/InterestGroupMemberList.tsx

```diff
@@ -1,43 +1,36 @@
 import sortBy from 'lodash/sortBy';
 import { Component } from 'react';
 import { Link } from 'react-router-dom';
+import TextInput from 'app/components/Form/TextInput';
 import Icon from 'app/components/Icon';
 import { ProfilePicture } from 'app/components/Image';
 import { Flex } from 'app/components/Layout';
 import Modal from 'app/components/Modal';
 import Tooltip from 'app/components/Tooltip';
+import shared from 'app/components/UserAttendance/AttendanceModal.css';
 import type { User, GroupMembership } from 'app/models';
 import styles from './InterestGroupMemberList.css';
 import type { ReactNode } from 'react';
 
-const ExtraInfo = ({ user, role }: { user: User; role: string }) => {
+const Name = ({ user, role }: { user: User; role: string }) => {
   if (role === 'member') {
     return <span>{user.fullName}</span>;
   }
 
-  let title: string;
   let roleStyle;
   switch (role) {
     case 'leader':
-      title = '(Leder)';
       roleStyle = styles.leader;
       break;
 
     case 'co_leader':
-      title = '(Nestleder)';
       roleStyle = styles.coleader;
       break;
 
     default:
       break;
   }
-
-  return (
-    <span>
-      <span className={roleStyle}>{user.fullName} </span>
-      <span className={styles.suffix}>{title}</span>
```

Comment:
I found this suffix to be redundant. First of all, it's in the tooltip, but the leaders have a gold shadow around their name and and gold star next to it. Also, their name is in the list of "leaders" next to the modal trigger.